GillianCore/GIL_Parser/GIL_Lexer.mll

```diff
@@ -110,7 +110,7 @@
       "assert",       GIL_Parser.ASSERT;
       "assume",       GIL_Parser.ASSUME;
       "assume_type",  GIL_Parser.ASSUME_TYPE;
-      "spec_var",     GIL_Parser.SPEC_VAR;
+      "symbol",       GIL_Parser.SYMBOL;
```

Comment:
It's a symbolic variable - I'm more for sth along the lines of `fresh_svar`.

Reply:
fresh_svar it is